### ISSUE TITLE
Show all open issues across all projects or organisations

### ISSUE THREAD

**Issue by obbardc** [gogs/gogs#5807]
**Is your feature request related to a problem? Please describe.**
I use gogs to manage multiple projects, each has a number of issues or feature requests.

**Describe the solution you'd like**
I would like to see at a glance a list of all open issues, similar to the list shown under `Issues > Assigned to you`.

**Describe alternatives you've considered**

**Additional context**

**Comment by unknwon** [comment]
Thanks for your feedback!

>at a glance a list of all open issues

By "all", do you mean all issues in your Gogs instance or what?

**Comment by obbardc** [comment]
Exactly, so I use issues to know what I have to do or fix, like an overall backlog.

Currently I setup an organisation per project, we can see per project by going to `Issues > Switch to organisation`.

Maybe it could work with a new menu item under `Switch Dashboard Context`?

**Comment by unknwon** [comment]
>Exactly, so I use issues to know what I have to do or fix, like an overall backlog.

Ah, OK. I should rephrase it again: all issues of repositories you have access to, not all issues from the instance, because that sounds like an admin feature :)

**Comment by obbardc** [comment]
> all issues of repositories you have access to, not all issues from the instance, because that sounds like an admin feature :)

Yes, that would work :-)!
